packages/realm-server/tests/realm-server-test.ts

```diff
@@ -1855,6 +1855,15 @@ module('Realm Server', function (hooks) {
           'embedded html looks correct (CardDef template)',
         );
 
+        // TODO: why is card-api module + css not present in the boxel index table?
+        assert.deepEqual(
+          json.data[0].relationships['prerendered-card-css'].data[0],
```

Comment:
This is expected to have css for `https://cardstack.com/base/card-api` but it doesn't since the module and css entry is not in the index table